modules/public/api.py

```diff
@@ -53,20 +57,25 @@
 }
 
 
-class InvalidResponseContent(Exception):
-    pass
-
-
-class ApiErrorResponse(Exception):
-    pass
+def process_response(response):
```

Comment:
Is every `get` going to go through `process_response`? If so, you might as well roll all these checks into `get` - although you could rename it if you feel the name doesn't quite suite any more.

Reply:
@nottrobin 
- I prefer not to put this code in the get function yet. Because it could break the api calls for the publisher pages. This should be done when I'll work on the refactor of the publishers.

- the name seems good to me for now, you thought about something else?

Reply:
Makes sense